test/e2e/edge-api-endpoints-can-receive-body/app/pages/api/edge.js

```diff
@@ -0,0 +1,11 @@
+export default async (req) => {
+  if (!req.body) {
+    return new Response('Body is required', { status: 400 })
+  }
+
+  return new Response(`got: ${await req.text()}`)
```

Comment:
Should there also be a test for `req.json()`?

Reply:
It's an implementation details: I just wanted to make sure we get the data, using `req.text()` will read the body in its entirety and it's implemented in edge-runtime (practically using undici)

So `req.text()`, `req.json()` and `req.arrayBuffer()` are all the same, but one thing we might want to test is streaming content into the edge function